packages/amplify-cli/package.json

```diff
@@ -47,6 +47,10 @@
     "@aws-amplify/amplify-category-predictions": "5.0.1",
     "@aws-amplify/amplify-category-storage": "3.7.4",
     "@aws-amplify/amplify-cli-logger": "1.3.0",
+    "@aws-amplify/amplify-cli-shared-interfaces": "1.1.2",
+    "@aws-amplify/amplify-console-hosting": "2.4.4",
+    "@aws-amplify/amplify-container-hosting": "2.5.14",
+    "@aws-amplify/amplify-dotnet-function-template-provider": "2.4.6",
```

Comment:
This got lost on recent release merge.
![image](https://user-images.githubusercontent.com/5849952/222222933-b97b2b0a-5dc5-42b1-9c42-b048f20754dd.png)

Reply:
😢 Thanks for bringing this back